project-keeper/src/test/resources/com/exasol/projectkeeper/mavenrepo/json.json

```diff
@@ -0,0 +1,9 @@
+{"response": {
```

Comment:
Rename file to e.g. maven-central-version-response.json

Reply:
Moved file  to `test/resources/simulated-maven-central-version-response.json`